src/components/DecisioningEngine/createOnResponseHandler.js

```diff
@@ -30,6 +31,11 @@ export default ({
       response.getPayloadsByType(PERSONALIZATION_DECISIONS_HANDLE)
     );
 
+    // only evaluate events that include a personalization query
+    if (!eventContent.query) {
+      return;
+    }
+
```

Comment:
I think it would be better to include a method on event "hasQuery". I'm not sure if this test is unique to personalization requests though, I think we also add something to query for ECID.